rdr_service/offline/genomic_pipeline.py

```diff
@@ -234,6 +234,7 @@ def gem_a2_manifest_workflow():
     """
     with GenomicJobController(GenomicJob.GEM_A2_MANIFEST,
                               bucket_name=config.GENOMIC_GEM_BUCKET_NAME) as controller:
+        controller.sub_folder_name = config.GENOMIC_GEM_A2_MANIFEST_SUBFOLDER
```

Comment:
Thanks for finding this. Can we just set it as a parameter in the controller when it's being invoked?

Reply:
Yes, good suggestion. I pushed a new commit that implements this.